### ISSUE TITLE
The cards are not perfectly aligned

### ISSUE THREAD

**Issue by divyansharma001** [swapagarwal/swag-for-dev#1275]
## Issue regarding the website

- [ ] I've checked that this isn't a duplicate issue.

The images are not perfectly aligned. This last card will look better in the left corner. Can you please assign this to me?

![image](https://github.com/user-attachments/assets/48184710-f5e6-4796-b274-f42cb63ac931)

**Comment by Tejas-Sharma-dev** [comment]
Yes, the issue is not position, but card alignment, card height is depended on swag description. we can have limited char with "read more" functionality.

**Comment by aslafy-z** [comment]
Related #1254

**Comment by bhavy3103** [comment]
I'm interested in solving this issue.

**Comment by Jai2082001** [comment]
Hi I am Interested in Solving the Issue
